src/apis/avm/api.ts

```diff
@@ -874,8 +876,8 @@ export class AVMAPI extends JRPCAPI {
     throw new Error("Error - AVMAPI.buildImportTx: Invalid destinationChain type: " + (typeof sourceChain) );
   }
   
-  const avmUTXOResponse: AVMUTXOResponse = await this.getUTXOs(ownerAddresses, srcChain, 0, undefined)
-  const atomicUTXOs: UTXOSet = await avmUTXOResponse.utxos;
+  const avmUTXOResponse: iAVMUTXOResponse = await this.getUTXOs(ownerAddresses, srcChain, 0, undefined)
```

Comment:
I like the use of `i` to denote an interface, is that in our style guide?

Reply:
Yes I added it to [our style guide](https://ava-labs.atlassian.net/wiki/spaces/HYD/pages/1243447307/AvalancheJS+Style+Guide#Interfaces) last week. Nice catch!